### PR TITLE
Adds Extra labels support for imported series.

### PR DESCRIPTION
**Pull request by f41gh7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,15 +15,19 @@ Features:
    * [Data mapping](#data-mapping)
    * [Configuration](#configuration)
    * [Filtering](#filtering)
+   * [ExtraLabels](#adding-extra-labels)
```
Comment (hagen1778) [inline README.md:18]: nit: i think, having "ExtraLabels" link on level-1 of this link tree if preferable - no need to repeat for each mode.

File: vm/vm.go [hunk 2]
```diff
@@ -74,6 +76,21 @@ func (im *Importer) Stats() string {
 	return im.s.String()
 }
 
+// AddExtraLabelsToImportPath - adds extra labels query params to given url path.
+func AddExtraLabelsToImportPath(path string, extraLabels []string) (string, error) {
```
Comment (hagen1778) [inline vm/vm.go:80]: nit: would be great to have test for this func

**Review by hagen1778 — APPROVED** [review]

File: vm/vm.go (hunk 2 above)
Comment (f41gh7) [reply vm/vm.go:80]: added test

File: README.md (hunk 1 above)
Comment (f41gh7) [reply README.md]: updated.

**Comment by valyala** [comment]
See also https://github.com/VictoriaMetrics/vmctl/issues/34

**Review by hagen1778 — APPROVED** [review]

**Comment by hagen1778** [comment]
Thanks @f41gh7!

**Pull request merged**